text/0000-move-analyze-phase.md

```diff
@@ -0,0 +1,90 @@
+# Meta
+[meta]: #meta
+- Name: Move analyze phase
+- Start Date: 2021-01-13
+- Author(s): [@jkutner](github.com/jkutner/) [@jabrown85](github.com/jabrown85)
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: [buildpacks/spec#172](https://github.com/buildpacks/spec/pull/172)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+This is a proposal to re-order and adjust Lifecycle phases. Specifically, moving "analyze" before "detect".
+
+# Motivation
+[motivation]: #motivation
+
+Doing this would support the following features and capabilities:
+* [Stack buildpacks](https://github.com/buildpacks/rfcs/pull/111), which require a phase to read run-image mixins validation prior to detection
+* Validating registry access for all images that are used can happen prior to `detector` or `builder` phases, providing faster failures for end users.
+
+# What it is
```

Comment:
Maybe it's worth adding in what Platform API we're going to introduce these changes.